Update documentaion and parameter schema (#452)
nf-core/funcscan: docs/usage.md

This will launch the pipeline with the `docker` configuration profile. See below for more information about profiles.

To run any of the three screening workflows (AMP, ARG, and/or BGC), taxonomic classification, and/or protein annotation, switch them on by adding the respective flag(s) to the command:

- `--run_amp_screening`
- `--run_arg_screening`
- `--run_bgc_screening`

> - `--run_taxa_classification` (for additional taxonomic annotations)

Reviewer: ```suggestion
- `--run_taxa_classification` (for optional additional taxonomic annotations)
```

> - `--run_protein_annotation` (for additional protein family and domain annotation)

Reviewer: ```suggestion
- `--run_protein_annotation` (for optional additional protein family and domain annotation)
```

  The contents of the directory should have files such as `<dbname>.version` and `<dbname>.taxonomy` in the top level.

> - An MMseqs2 ready database. These databases were compiled by the developers of MMseqs2 and can be called using their labels. All available options can be found [here](https://github.com/soedinglab/MMseqs2/wiki#downloading-databases). Only use those databases that have taxonomy files available (i.e. Taxonomy column shows "yes"). By default MMseqs2 in the pipeline uses '[Kalamari](https://github.com/lskatz/Kalamari)', and runs an aminoacid-based alignment. However, if the user requires a more comprehensive taxonomic classification, we recommend the use of [GTDB](https://gtdb.ecogenomic.org/), but for that please remember to increase the memory, CPU threads and time required for the process `MMSEQS_TAXONOMY`.

Reviewer: ```suggestion
- An MMseqs2 ready database. These databases were compiled by the developers of MMseqs2 and can be called using their labels. All available options can be found [here](https://github.com/soedinglab/MMseqs2/wiki#downloading-databases). Only use those databases that have taxonomy files available (i.e. Taxonomy column shows "yes"). By default MMseqs2 in the pipeline uses '[Kalamari](https://github.com/lskatz/Kalamari)', and runs an amino acid-based alignment. However, if the user requires a more comprehensive taxonomic classification, we recommend the use of [GTDB](https://gtdb.ecogenomic.org/), but for that please remember to increase the memory, CPU threads and time required for the process `MMSEQS_TAXONOMY`.
```

```

:::note{.fa-whale}

> For both [DRAMP](http://dramp.cpu-bioinfor.org/) and [APD](https://aps.unmc.edu/), AMPcombi removes entries that contain any non-amino-acid residues by default.

Reviewer: ```suggestion
For both [DRAMP](http://dramp.cpu-bioinfor.org/) and [APD](https://aps.unmc.edu/), AMPcombi removes entries that contain any non-amino acid residues by default.
```
